src/web/app/components/preview-session-panel/preview-session-panel.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="card-body background-color-light-blue text-center">
+  <div class="row align-items-center">
+    <div class="col-lg-2 text-lg-right">
+      <b>Preview Session:</b>
+    </div>
+    <div class="col-lg-5">
+      <select id="preview-student" class="form-control preview-select" [(ngModel)]="emailOfStudentToPreview">
+        <option *ngFor="let student of studentsOfCourse" [ngValue]="student.email">[{{ student.teamName }}] {{ student.name }}</option>
+      </select>
+      <a *ngIf="emailOfStudentToPreview.length !== 0; else previewStudentBtn" routerLink="/web/sessions/submission"
+         [queryParams]="{ courseid: courseId, fsname: feedbackSessionName, previewas: emailOfStudentToPreview }"
+         target="_blank"
+         rel="noopener noreferrer"
+         [ngbTooltip]="emailOfStudentToPreview.length !== 0 ? 'View how this session would look like to a student who is submitting feedback.' : 'Preview is unavailable if the course has yet to have any student enrolled.'">
+        <ng-container *ngTemplateOutlet="previewStudentBtn"></ng-container>
+      </a>
+      <ng-template #previewStudentBtn>
+        <button id="btn-preview-student" class="btn btn-primary" [disabled]="emailOfStudentToPreview.length === 0 || forDisplayOnly">Preview as Student</button>
+      </ng-template>
+    </div>
+    <div class="col-lg-5">
+      <select id="preview-instructor" class="form-control preview-select" [(ngModel)]="emailOfInstructorToPreview">
+        <option *ngFor="let instructor of instructorsCanBePreviewedAs" [ngValue]="instructor.email">{{ instructor.name }}</option>
+      </select>
+      <a *ngIf="emailOfInstructorToPreview.length !== 0; else previewInstructorBtn" routerLink="/web/instructor/sessions/submission"
+         [queryParams]="{ courseid: courseId, fsname: feedbackSessionName, previewas: emailOfInstructorToPreview }"
+         target="_blank"
+         rel="noopener noreferrer"
+         ngbTooltip="View how this session would look like to an instructor who is submitting feedback. Only instructors with submit privileges are included in the list.">
+        <ng-container *ngTemplateOutlet="previewInstructorBtn"></ng-container>
+      </a>
+      <ng-template #previewInstructorBtn>
+        <button id="btn-preview-instructor" class="btn btn-primary" [disabled]="emailOfInstructorToPreview.length === 0 || forDisplayOnly">Preview as Instructor</button>
+      </ng-template>
+    </div>
+  </div>
```

Comment:
You very likely miss one closing `</div>` here

Reply:
fixed in 7c8e31a